lib/rouge/lexers/hocon.rb

```diff
@@ -0,0 +1,60 @@
+# -*- coding: utf-8 -*- #
+# frozen_string_literal: true
+
+module Rouge
+  module Lexers
+    load_lexer 'json.rb'
+
+    class HOCON < JSON
+      title 'HOCON'
+      desc "Human-Optimized Object Configuration Notation (https://github.com/lightbend/config)"
+      tag 'hocon'
+      filenames '*.hocon'
+
+      prepend :root do
+        # Comments
+        rule %r(//.*?$), Comment::Single
+        rule %r(#.*?$), Comment::Single
+
+        # Interpolation
+        rule %r/[$][{][?]?/, Literal::String::Interpol, :interpolation
+
+        # Strings
+        rule %r/"""/, Literal::String::Double, :multiline_string
+
+        # Keywords
+        rule %r/\b(?:include|url|file|classpath)\b/, Keyword
+
+        # Symbols
+        rule %r/[()={},:\[\]]/, Punctuation
+
+        # Keys
+        rule %r/([\w\-\.]+? ?)([{:=]|\+=)/ do
```

Comment:
Can there really be only one optional space here? Something like `foo   = 'bar'` is no good?

Reply:
Oops, good catch, I've changed this to zero-or-more spaces.